openreview/venue/matching.py

```diff
@@ -1165,6 +1165,23 @@ def deploy_assignments(self, assignment_title, overwrite):
             reviewer_name = venue.area_chairs_name
             review_name = 'Meta_Review'
 
+        # Remove reviewers_proposed_assignment_title if deploying reviewer assignments
+        if reviewer_name == venue.reviewers_name:
+            venue_group = client.get_group(venue.id)
+            if 'reviewers_proposed_assignment_title' in venue_group.content:
```

Comment:
I don't think it is needed to check if the property exists to set the delete true, can you try just posting the edit?